### ISSUE TITLE
Kodi thumbnail metadata not cleaned up when replacing episode release

### ISSUE THREAD

**Issue by aMvEL2** [SickGear/SickGear#691]
BRANCH: develop / COMMIT: 8432a313f02a2975e4b16e3f000bd7c6d263accb

**Issue:**
The -thumb episode metadata is not removed when replacing a release of an episode with another one.

**Reprodouction:**
1: Make sure to have thumbnail generation for Kodi metadata enabled.
(Post-processing -> Metadata -> Kodi -> Episode thumbnails).
2: Postprocess an initial release, and check that the -thumb.jpg file has been created for that episode in its show-dir. (I used show Orphan Black)
3: Postprocess a replacement release, i.e. quality upgrade from SD to 720p.
4: See that the -thumb.jpg for the old release remains in the show directory.

The log only mentions that it deletes the old episode file and .nfo file.

**Log:**
http://pastebin.com/n8tS1X4t

I don't know when this problem started, as I only started using -thumb-files recently.

**Comment by aMvEL2** [comment]
After having checked the code, I believe the error happens in method "list_associated_files" (Line 157 in postProcessor.py).

This method does only check for associated files which has the same base name and different variants of file extensions, not files which has an extended base name.

If I added it manually to the list of associated files, it correctly deletes the orphan thumb-files.


Sadly my programming skills are shit, and I just used 2hrs to figure out how to read the code..
